pages/index.js

```diff
@@ -67,6 +87,22 @@ export default function HomePage() {
 
   return (
     <StyledDiv>
+      <button /* Only for testing of success-marker for past days,
+        it creates the saved daily entry for yesterday.
+        Clicking with default local storage values will mark yesterday either
+        red or green depending on the currently displayed calorie count.
+        The history with entries is also logged in the console.*/
```

Comment:
I don't understand why you want to do this, could you elaborate on that, please?

Reply:
to prove the functionality of marking a date in the calendar without waiting for midnight